src/os/ui/ogc/iogcdescriptor.js

```diff
@@ -157,6 +157,19 @@ os.ui.ogc.IOGCDescriptor.prototype.getWmsParams;
 os.ui.ogc.IOGCDescriptor.prototype.setWmsParams;
 
 
+/**
+ * Get the additional WFS parameters to include in requests
+ * @return {?goog.Uri.QueryData}
+ */
+os.ui.ogc.IOGCDescriptor.prototype.getWfsParams;
```

Comment:
`os.ui.ogc.OGCDescriptor` should implement these as well. It's only used by tests so the compiler didn't catch it. We may also want to move it out of `src` at some point, though not necessary here.

Reply:
Done.